src/main/ipc/updater.ts

```diff
@@ -66,7 +67,8 @@ async function github(
     };
   }
 
-  const asset = res.data.assets.find((asset) => asset.name.endsWith(".asar"));
+  const asset = res.data.assets.find((asset) => asset.name === path);
```

Comment:
I think you should use the ID instead since the path could change.

Reply:
wdym ID? the path is just the id with .asar added at the end

Reply:
The path is the file name, which doesn't necessarily have to be the ID.